internal/cluster/clustersetup/activity_install_ingress_controller.go

```diff
@@ -145,7 +162,19 @@ func (i *IngressControllerPostHook) Do(cluster CommonCluster) error {
 
 	namespace := global.Config.Cluster.Namespace
 
-	return i.helmService.ApplyDeployment(context.Background(), cluster.GetID(), namespace, config.Ingress.Chart, "ingress", valuesBytes, config.Ingress.Version)
+	err = a.helmService.ApplyDeployment(
```

Comment:
**Question**: are we using simple apply without reusing chart values here because we believe ingress is not deployed at this point opposed to for example the cluster autoscaler solution?